src/redux/features/admin/activateUserAcountSlice.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable prettier/prettier */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+import API from '@/api/api';
+import isAuth from '@/helpers/isAuth';
+
+export const activateUserAcount = createAsyncThunk(
+  'activate/user',
+  async (data: string) => {
+    const body={
+      email:data,
+    }
+    const { data: activateUser } = await API.patch('/users/activate',body, {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+    return activateUser;
```

Comment:
Use a try-catch block to catch any exceptions that might occur during the API call and handle them accordingly.

hint:
```typescript
try {
      const body = {
        email: email,
      };

      const response = await API.patch('/users/activate', body, {
        headers: {
          'Content-Type': 'application/json',
        },
      });

      const activateUser = response.data;
      return activateUser;
    } catch (error) {
      // Handle the error (e.g., logging, error message, etc.)
      throw new Error('Failed to activate user account');
    }
```